.github/workflows/release.yaml

```diff
@@ -237,13 +237,13 @@ jobs:
         include:
           - os: linux
             arch: amd64
-            target: archive
+            target: archive_novulkan
```

Comment:
would this revert to `archive` later?

Reply:
Yes.  Once we've fixed up the last few bugs in Vulkan support we can simply revert this change.